src/mpyl/project_execution.py

```diff
@@ -0,0 +1,22 @@
+"""This module contains the ProjectExecution class."""
+import uuid
+from dataclasses import dataclass
+
+from .project import Project
+
+
+@dataclass(frozen=True)
+class ProjectExecution:
+    project: Project
+    cache_key: str
+    cached: bool
+
+    @staticmethod
+    def always_run(project: Project):
```

Comment:
I see this is only used in unit tests.. Can we move them to a test resources helper class? so move to tests/ from src/

Reply:
It's also used for manual selection

Reply:
ah okay. I would add a docstring to this method then